scripts/infrastructure/destroy/destroy-corrupted-workspace.sh

```diff
@@ -22,7 +22,7 @@ function _destroy_corrupted_workspace() {
         return 1
     fi
 
-    dev_acct=$(_get_aws_account_id "$ENV")
+    dev_acct=$(_get_aws_account_id "$ENV") # TODO Broken by PI-165
```

Comment:
Is this going to be fixed in this ticket? If not have you writtten up another ticket for it?

Reply:
think i've "fixed" this

Reply:
Ah yeah, it was a placeholder because I broke the function and missed the correction.  Juggling too many things.